docs/sources/datasources/influxdb/configure-influxdb-data-source/_index.md

```diff
@@ -0,0 +1,260 @@
+---
+aliases:
+  - ../data-sources/influxdb/
+  - ../data-sources/influxdb/provision-influxdb/
+  - ../features/datasources/influxdb/
+  - provision-influxdb/
+description: Guide for using InfluxDB in Grafana
+keywords:
+  - grafana
+  - influxdb
+  - guide
+  - flux
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Configure the InfluxDB data source
+title: Configure the InfluxDB data source
+weight: 300
+refs:
+  build-dashboards:
+  provision-grafana:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/provisioning/#provision-grafana
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/provisioning/#provision-grafana  
+---
+
+# Configure the InfluxDB data source
+
+This document describes how to configure the InfluxDB data source and reviews configuration options. 
```

Comment:
I am not familiar with the "voice" of documentation but wondering if a more active voice works better. `This document provides instructions for configuring the InfluxDB data source and explains the available configuration options.`

Reply:
I like it!